pkg/k8s/labels.go

```diff
@@ -126,6 +126,9 @@ const (
 	// ProxyConfigAnnotationsPrefix is the prefix of all config-related annotations
 	ProxyConfigAnnotationsPrefix = "config.linkerd.io"
 
+	// VizAnnotationPrefix is the prefix for all viz related annotations
+	VizAnnotationPrefix = "viz.linkerd.io"
```

Comment:
This and `ProxyDisableTapAnnotation` are specific to the Viz extension, so I don't think they belong as part of the Linkerd's `pkg` package.

In #5651, the Viz package had it's own labels added so this can removed, and `ProxyDisableTapAnnotation` should move to that file.